construction/data/res_config_settings.xml

```diff
@@ -1,23 +1,191 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<odoo noupdate="1">
-    <record id="res_config_account_setting" model="res.config.settings">
-        <field name="group_project_stages" eval="1"/>
-        <field name="group_discount_per_so_line" eval="1"/>
-        <field name="group_product_pricelist" eval="1"/>
-        <field name="group_warning_sale" eval="1"/>
-        <field name="group_warning_purchase" eval="1"/>
-        <field name="invoiced_timesheet" eval="'approved'"/> 
-        <field name="group_warning_stock" eval="1"/> 
-        <field name="group_stock_reception_report" eval="1"/> 
-        <field name="group_sale_delivery_address" eval="1"/>
-        <field name="documents_product_settings" eval="1"/>
-        <field name="documents_hr_settings" eval="1"/>
-        <field name="group_analytic_accounting" eval="1"/>
-        <field name="group_uom" eval="1"/>
-        <field name="group_sale_order_template" eval="1"/>
-    </record>
-
-    <function name="execute" model="res.config.settings">
-        <value eval="[ref('res_config_account_setting')]"/>
-    </function>
+<odoo>
+    <record id="x_setting_field_progress_management" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="field_description">Progress Management</field>
+        <field name="model_id" ref="base_setup.model_res_config_settings"/>
+        <field name="name">x_setting_progress_management</field>
+        <field name="readonly" eval="False"/>
+        <field name="store" eval="True"/>
+        <field name="compute"><![CDATA[
+self['x_setting_progress_management'] = self.env['ir.config_parameter'].sudo().get_param('construction_configuration.enable_progress_management', '0') != '0' or bool(self.env['ir.module.module'].search_count([('name', '=', 'construction_progress_management')], limit=1))
```

Comment:
maybe wrong if the module is uninstalled from another mean (like from Apps). Neither the config param nor the setting will be updated :-/